db/schema.rb

```diff
@@ -200,7 +200,7 @@
   end
 
   create_table "school_cohorts", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
-    t.string "induction_programme_choice", default: "not_yet_known", null: false
+    t.string "induction_programme_choice", null: false
```

Comment:
Is that gonna work? Can we make it nullable?

Reply:
Why wouldn't it work? The idea is you can't do anything else until you've decided this

Reply:
If it's null, everything else breaks

Reply:
I hope you don't read it for a while, but I think I got confused and thought `SchoolCohorts` would exist before the user selects anything - which would obviously break things. But I don't think they will.